audit/projects/k8s-staging-apisnoop/services/enabled.json

```diff
@@ -49,6 +49,7 @@
         "staging-composer.sandbox.googleapis.com",
         "staging-managedvms.googleapis.com",
         "test-appengineflex.sandbox.googleapis.com",
+        "test-cloudfunctions.sandbox.googleapis.com",
```

Comment:
I wonder how valuable this file really is? Or maybe just switching from JSON to normal output would be better?  e.g.

```
NAME                                     TITLE
bigquery.googleapis.com                  BigQuery API
bigquery.googleapis.com                  BigQuery API
bigquerystorage.googleapis.com           BigQuery Storage API
cloudkms.googleapis.com                  Cloud Key Management Service (KMS) API
cloudmonitoring.googleapis.com           Cloud Monitoring API
compute.googleapis.com                   Compute Engine API
computescanning.googleapis.com           Compute Scanning API
container.googleapis.com                 Kubernetes Engine API
containeranalysis.googleapis.com         Container Analysis API
containerregistry.googleapis.com         Container Registry API
containerscanning.googleapis.com         Container Scanning API
dataflow.googleapis.com                  Dataflow API
dataproc-control.googleapis.com          Cloud Dataproc Control API
dataproc.googleapis.com                  Cloud Dataproc API
datastore.googleapis.com                 Cloud Datastore API
deploymentmanager.googleapis.com         Cloud Deployment Manager V2 API
dns.googleapis.com                       Cloud DNS API
logging.googleapis.com                   Cloud Logging API
monitoring.googleapis.com                Cloud Monitoring API
oslogin.googleapis.com                   Cloud OS Login API
pubsub.googleapis.com                    Cloud Pub/Sub API
replicapool.googleapis.com               Compute Engine Instance Group Manager API
replicapoolupdater.googleapis.com        Compute Engine Instance Group Updater API
resourceviews.googleapis.com             Compute Engine Instance Groups API
servicenetworking.googleapis.com         Service Networking API
sql-component.googleapis.com             Cloud SQL
sqladmin.googleapis.com                  Cloud SQL Admin API
stackdriver.googleapis.com               Stackdriver API
stackdriverprovisioning.googleapis.com   Stackdriver Provisioning Service
staging-cloudkms.sandbox.googleapis.com  Cloud Key Management Service (KMS) API
storage-api.googleapis.com               Google Cloud Storage JSON API
storage-component.googleapis.com         Cloud Storage
```

Reply:
It is much more human readable imho, which is the intent here.

Reply:
I aggree

Reply:
But lets change it in the next PR